pkg/datasource/csvsource/bybit_tick_downloader.go

```diff
@@ -0,0 +1,101 @@
+package csvsource
+
+import (
+	"bytes"
+	"compress/gzip"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+)
+
+func Download(symbol string, start time.Time) {
+	for {
+		var (
+			path     = fmt.Sprintf("pkg/datasource/csv/testdata/bybit/%s/", symbol)
+			fileName = fmt.Sprintf("%s%s.csv", strings.ToUpper(symbol), start.Format("2006-01-02"))
+		)
+
+		if fileExists(path + fileName) {
```

Comment:
use filepath.Join for the path concatenation

Reply:
yes true